doc/source/parameters/unresolved_cfd-dem/cfd_dem.rst

```diff
@@ -17,6 +17,8 @@ This subsection includes parameters related to multiphase flow simulations using
       set coupling frequency = 200
       set inlet boundary id = 2
       set outlet boundary id = 3
+      set implicit stabilization = false
```

Comment:
I think it's better to have the default here at true. And if we want more robustness, we can change it to false :). Let's keep the previous behavior as is and when we voluntarily face challenge, we put this to false :).